docs/development/local.md

```diff
@@ -0,0 +1,170 @@
+ * [minikube](./develop-prerequisites.md#minikube)
+ * [K8S deployment prerequisites](./install-prerequisites.md#kubernetes-deployment-prerequisites)
+ * in case you are planning to develop/contribute code, you should check the [local development prerequisites](./develop-prerequisites#local-development-prerequisites)
+
+#### Installation 
+
+Follow [run on Kubernetes](./../install/sm.md#run-on-Kubernertes) installation steps.
+
+### Service Manager on `cfdev/pcfdev`
+
+#### Prerequisites
+
+ * [general deployment prerequisites](./install-prerequisites.md#general-deployment-prerequisites)
+ * [local CF installation](./develop-prerequisites.md#local-cf-setup)
+ * [CF deployment prerequisites](./install-prerequisites.md#cloud-foundry-deployment-prerequisites)
+ * in case you are planning to develop/contribute code, you should check the [local development prerequisites](./develop-prerequisites#local-development-prerequisites)
+ 
+#### Configuration Adjustments
+
+Edit the following `env` vars in the `$GOPATH/src/github.com/Peripli/service-manager/deployment/cf/manifest.yml`:
+* set the token endpoint of the authorization server that was setup as part of the prerequisites
+    ```yml
+    env:
+      API_TOKEN_ISSUER_URL: https://uaa.dev.cfdev.sh
+    ```
+* set the storage configuration
+
+    If you are using an external database that is running on `localhost:5432` then set the `STORAGE_URI` (use 10.0.2.2 as the application will run inside the CF Dev VM and `10.0.2.2` maps to the host's `localhost`):
+
+    **Note:** You might need to setup a loopback alias. On MacOS, you may use `sudo ifconfig lo0 alias 10.0.2.2`.
+
+    ```yml
+    env:
+      STORAGE_URI: postgres://postgres:postgres@10.0.2.2:5432/postgres?sslmode=disable
+    ```
+
+    If your CF installation has a `postgres` service, then create and bind it to the application and set `STORAGE_NAME` to the postgres service instance name:
+
+    ```yml
+    env:
+      STORAGE_NAME: postgres
+    ```
+
+**Note:** Comment the `docker.image` in the `manifest.yml` in order to use local sources.
+
+#### Installation
+
+Follow [run on CF](./../install/sm.md#run-on-CF) installation steps.
+
+### Service Manager on `localhost`
+
+#### Prerequisites
+
+* check the [local development prerequisites](./develop-prerequisites.md#local-development-prerequisites) 
+
+#### Configuration Adjustments
+
+Edit the following `properties` in the `$GOPATH/src/github.com/Peripli/service-manager/deployment/cf/manifest.yml`:
+* set the token endpoint of the authorization server that was setup as part of the prerequisites
+    ```yml
+    api:
+      token_issuer_url: https://uaa.dev.cfdev.sh
+      skip_ssl_verification: true
+    ```
+* set the storage configuration
+
+    ```yml
+    storage:
+      uri: postgres://postgres:postgres@10.0.2.2:5432/postgres?sslmode=disable
+    ```
+
+    **Note:** Comment `storage.name` if its present in application.yml
+
+
+#### Installation
+
+```console
+$ go run main.go
```

Comment:
optionally provide command line arguments instead of modifying the keys in the above files.